zerver/webhooks/hellosign/doc.md

```diff
@@ -1,17 +1,33 @@
+# Zulip HelloSign integration
+
 Configuring the HelloSign integration is easy!
 
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
 
 1. Go to your HelloSign account **Settings**. Select **Integrations** and
    click on **API**.
 
 1. Set **Account Callback** to the URL constructed above. Click **Test**
```

Comment:
We can use "generated" instead of "constructed" for the URL, across all the docs, to reflect the change that URLs are now generated.

Reply:
Done!